### ISSUE TITLE
Prerequisites: iptables must be nf_table version, not legacy

### ISSUE THREAD

**Issue by stephan-henningsen** [docker/docs#18418]
### Is this a docs issue?

- [X] My issue is about the documentation content or website

### Type of issue

There's a problem with the website

### Description

I think the full requirement for `iptables` should be stated in the docs, because it's not enough to have `iptables` installed in that version or newer.  It has to have a special option enabled at compile time, otherwise docker won't fully work.

I've detailed the problem here: https://github.com/moby/moby/issues/46636

Also, I think the cgroups requirement is a bit out-dated, since most distros use v2 which is in fact a hierarchy on a single mount point.

### Location

https://docs.docker.com/engine/install/binaries/#prerequisites

### Suggestion

`iptables` version 1.4 or higher, built for nf_tables (not legacy)

or:

`iptables` version 1.4 or higher, built for nf_tables (not legacy, check with `iptables --version`)

**Comment by neersighted** [comment]
As per my comment in https://github.com/moby/moby/issues/46636#issuecomment-1760978986, I think this is both incorrect and actively counterproductive. Given https://github.com/moby/moby/issues/39590, if we are to mention an iptables variant we should encourage use of iptables-legacy; or at the very least we should advise users to ensure they only use one netfilter backend for firewall rules.

**Comment by dvdksn** [comment]
cc @akerouanton

**Comment by neersighted** [comment]
I'm going to close this for now; we can open a new ticket suggesting iptables-legacy if we decide that is necessary advice.

**Comment by stephan-henningsen** [comment]
I agree.
